app/controllers/registrations_controller.rb

```diff
@@ -57,6 +57,6 @@ def callback
   private
 
   def registration_params
-    params.require(:registration).permit(:username)
+    params.require(:registration).permit(:username, :nickname)
```

Comment:
Hi @jlduran,

Good call, that warning's been bothering for a while...
I understand this remove the warning, however, we don't really need to permit the `:nickname` because we're not consuming it.

Now that I look at this I am wondering why we have this param sanitization when we're not doing any mass assignment anywhere.

I think what we want might be

```diff
diff --git a/app/controllers/registrations_controller.rb b/app/controllers/registrations_controller.rb
index 31a5f8f..c36091c 100644
--- a/app/controllers/registrations_controller.rb
+++ b/app/controllers/registrations_controller.rb
@@ -5,11 +5,11 @@ class RegistrationsController < ApplicationController
   end
 
   def create
-    user = User.new(username: registration_params[:username])
+    user = User.new(username: params[:registration][:username])
 
     create_options = WebAuthn::Credential.options_for_create(
       user: {
-        name: registration_params[:username],
+        name: params[:registration][:username],
         id: user.webauthn_id
       }
     )
@@ -53,10 +53,4 @@ class RegistrationsController < ApplicationController
       render json: "Verification failed: #{e.message}", status: :unprocessable_entity
     end
   end
-
-  private
-
-  def registration_params
-    params.require(:registration).permit(:username)
-  end
 end
```